README.md

```diff
@@ -2,145 +2,69 @@
-above, but apply a patched version of the operator resource that uses the newly
-built
-operator image:
-
-```shell
-oc patch --dry-run -f deploy/operator.yaml -o yaml -p \
-  '{"spec":{"template":{"spec":{"containers":[{"name": "smart-gateway-operator", "image": "'"$(oc registry info)/$(oc project -q)"'/smart-gateway-operator:dev"}]}}}}' \
-  | oc apply -f -
-... (See above)
-```
+<todo>
 
 ### FIXME - Unit testing
 
 With molecule (Currently broken)
 
 ### Integration testing
 
-Test the newly built operator with the Service Assurance Framework (SAF).
+Test the newly built operator with the Service Assurance Operator.
 
-The sections below can be iterated on during development, in brief:
-
-1. Clean out the project space
-1. Build the operator
-1. Import the operator to an ImageStream in the project
-1. Deploy SAF in the same project
-
-#### Prepare project
-
-Create the `sa-telemetry` project and push the operator image to an image stream
-in its registry namespace.
-
-```shell
-PROJECT=sa-telemetry
-oc delete project ${PROJECT}
-oc new-project ${PROJECT}
-SA_DOCKER_IMAGE="$(minishift openshift registry)/${PROJECT}/smart-gateway-operator:dev"
-operator-sdk build "${SA_DOCKER_IMAGE}"
-docker push "${SA_DOCKER_IMAGE}"
-oc import-image --insecure=true smart-gateway-operator:latest --from=$(oc registry info)/${PROJECT}/smart-gateway-operator:dev --confirm
-```
-
-**NOTE:** The `--insecure=true` flag is to work around an issue where the
-openshift internal registry is not trusted. It is expected we can drop it
-in a future revision. See [minishift issue](https://github.com/minishift/minishift/issues/2544)
-and [openshift issue](https://github.com/openshift/origin/issues/20604)
-
-#### Deploy SAF
-
-Follow the process for deploying SAF in a testing environment. Hints:
-
-* [Travis automation](https://github.com/redhat-service-assurance/telemetry-framework/blob/master/.travis.yml#L12)
-* [SAF Documentation](https://github.com/redhat-service-assurance/telemetry-framework/blob/master/deploy/README.md#quickstart-minishift)
-
-NOTE: This process will cause a couple of expected errors while running the
-`quickstart.sh` script in telemetry-framework:
-
-* `Error from server (AlreadyExists): project.project.openshift.io "sa-telemetry" already exists`
-  * It was created in order to stage the operator image
-* `error: the tag "latest" points to "172.30.1.1:5000/sa-telemetry/smart-gateway-operator" - use the 'tag' command if you want to change the source to "quay.io/redhat-service-assurance/smart-gateway-operator:latest`
-  * It was created in order to stage the operator image
-* `Error from server (AlreadyExists): error when creating "operators/smartgateway/crds/smartgateway_v1alpha1_smartgateway_crd.yaml": customresourcedefinitions.apiextensions.k8s.io "smartgateways.smartgateway.infra.watch" already exists`
-  * You may see this if you have been testing the operator prior to deploying it
-    for integration with SAF
+<todo>
```

Comment:
Not super happy with a patch that makes the documentation go backwards by so much.

Reply:
Yea sorry about that... got everything working, but we don't have full procedures. I will make a card to have it addressed and fixed in Sprint 22.